core/domain/state_domain.py

```diff
@@ -277,6 +277,37 @@ class InteractionInstance(python_utils.OBJECT):
     # The default interaction used for a new state.
     _DEFAULT_INTERACTION_ID = None
 
+    # List of interactions storing HTML in the rule spec inputs field.
```

Comment:
I think you're still missing the backend test that validates that these constants are what they should be. Think about it this way: if a developer adds a new interaction with an html field, the backend tests should fail if these constants are not updated. How can you write a backend test that ensures that?

Reply:
(Also, sorry if you've written this test already and I missed it. If that's the case could you please point me to it?)